moonshot/src/api/api_dataset.py

```diff
@@ -44,3 +45,36 @@ def api_get_all_datasets_name() -> list[str]:
     """
     datasets_name, _ = Dataset.get_available_items()
     return datasets_name
+
+
+def api_create_datasets(
+    name: str, description: str, reference: str, license: str, method: str, **kwargs
```

Comment:
what is this method:str

Reply:
whether to convert dataset from hugging face or csv